src/Native/Unix/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(CoreFX)
 set(CMAKE_MACOSX_RPATH ON)
 set(CMAKE_INSTALL_PREFIX $ENV{__CMakeBinDir})
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I guess this can be removed now too. There are also a few more CXX variables which are used and need replacement too:

- CMAKE_CXX_COMPILER_VERSION
- CMAKE_CXX_FLAGS_DEBUG
- CLR_SANITIZE_CXX_FLAGS